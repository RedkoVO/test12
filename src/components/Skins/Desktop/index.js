import React from 'react'
import PropTypes from 'prop-types'
import withStyles from '@material-ui/core/styles/withStyles'

import MenuNavigation from '../../MenuNavigation'
import HeaderCatalog from '../components/HeaderCatalog'
import TradeTop from '../components/TradeTop'
import TradeSkins from '../components/TradeSkins'
import Footer from '../../Footer/Desktop/'

import VisaLogo from '../../../assets/images/visa.png'
import BitcnpayLogo from '../../../assets/images/bitcnpay.png'
import AgspayLogo from '../../../assets/images/agspay.png'

import styles from './styles'

const Skins = ({ classes, handleLogout, isDesktop, skins }) => (
  <div className="mainWrapperTrade">
    <div className={classes.containerTradeWrap}>
      <div className={classes.containerTrade}>
        <header className={classes.headerSkins}>
          <MenuNavigation page="dark" handleLogout={handleLogout} />
          <HeaderCatalog isDesktop={isDesktop} />
        </header>

        <main className={classes.tradeMain}>
          <TradeTop />
          <TradeSkins skins={skins} />

          <div className="tradePayment">
            <div className="paymentTitle">Choose the way to cash out:</div>
            <div className="paymentWay">
              <div className="paymentWayItem paymentSelectItem">
                <div className="paymentWayItemImg">
                  <img src={VisaLogo} alt="" />
                </div>
                <div className="paymentWayItemName">Credit Card</div>
                <div className="paymentWayItemUnderline selected" />
              </div>
              <div className="paymentWayItem">
                <div className="paymentWayItemImg">
                  <img src={BitcnpayLogo} alt="" />
                </div>
                <div className="paymentWayItemName">Bitcoin</div>
                <div className="paymentWayItemUnderline" />
              </div>
              <div className="paymentWayItem">
                <div className="paymentWayItemImg">
                  <img src={AgspayLogo} alt="" />
                </div>
                <div className="paymentWayItemName">CryptaurCoin</div>
                <div className="paymentWayItemUnderline" />
              </div>
            </div>
            <div className="paymentCard">
              <div className="paymentCardTitle">Cashout on my Credit card:</div>
              <div className="paymentInputWrap">
                <input
                  type="text"
                  className="paymentCardNum"
                  placeholder="Card Number"
                />
              </div>
              <div className="paymentInputWrap2">
                <input
                  type="text"
                  className="paymentCardHolder"
                  placeholder="Card Holder"
                />
              </div>

              <a href="#" className="paymentCardMonth">
                MM
              </a>
              <a href="#" className="paymentCardYear">
                YYYY
              </a>
              <div className="paymentInputWrap3">
                <input
                  type="text"
                  className="paymentCardCvc"
                  placeholder="CVC"
                />
              </div>
            </div>
            <div className="paymentSelected">
              <div className="paymentSelectedTitle">You selected 6 items:</div>
              <div className="paymentSelectedItem">
                <div className="selectedText">1) Good EWeapon 1.2 ....</div>
                <div className="selectedPrice">$12.20</div>
                <a href="#" className="selectedClose" />
              </div>
              <div className="paymentSelectedItem">
                <div className="selectedText">1) Good EWeapon 1.2 ....</div>
                <div className="selectedPrice">$12.20</div>
                <a href="#" className="selectedClose" />
              </div>
              <div className="paymentSelectedItem">
                <div className="selectedText">1) Good EWeapon 1.2 ....</div>
                <div className="selectedPrice">$12.20</div>
                <a href="#" className="selectedClose" />
              </div>
              <div className="paymentSelectedItem">
                <div className="selectedText">1) Good EWeapon 1.2 ....</div>
                <div className="selectedPrice">$12.20</div>
                <a href="#" className="selectedClose" />
              </div>
              <div className="paymentSelectedItem">
                <div className="selectedText">1) Good EWeapon 1.2 ....</div>
                <div className="selectedPrice">$12.20</div>
                <a href="#" className="selectedClose" />
              </div>
              <div className="paymentSelectedItem">
                <div className="selectedText">1) Good EWeapon 1.2 ....</div>
                <div className="selectedPrice">$12.20</div>
                <a href="#" className="selectedClose" />
              </div>
            </div>
            <div className="paymentAmmount">
              <div className="paymentAmmountText">Total ammount: </div>
              <div className="paymentAmmountFill" />
              <div className="paymentAmmountPrice">$67.45</div>
            </div>
            <div className="paymentBonus">
              <input
                type="checkbox"
                className="paymentBonusCheck"
                id="paymentBonusCheck"
              />
              <label
                htmlFor="paymentBonusCheck"
                className="paymentBonusCheckLabel"
              >
                Yes, I have a bonus code!
              </label>
            </div>
            <div className="paymentInputWrap4">
              <input
                type="text"
                className="paymentBonusInput"
                placeholder="Enter Your Bonus Code"
              />
            </div>
            <a href="#" className="paymentBtn">
              CASHOUT
            </a>
          </div>
        </main>
      </div>
    </div>

    <Footer />
  </div>
)

Skins.propTypes = {
  classes: PropTypes.object,
  handleLogout: PropTypes.func,
  isDesktop: PropTypes.bool,
  skins: PropTypes.array
}

export default withStyles(styles)(Skins)
