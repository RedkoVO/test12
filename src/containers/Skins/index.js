import compose from 'recompose/compose'
import { withProps, pure } from 'recompose'

import AsyncSkinsDesktop from '../../components/Skins/Desktop/AsyncSkinsDesktop'

const tmpArrSkins = [
  {
    id: 1,
    title: 'Gun: Good Weapon2 1.2',
    cost: '$11.10 FN',
    discountsCost: '$12.30'
  },
  {
    id: 2,
    title: 'Gun: Good Weapon2 1.2',
    cost: '$11.10 FN',
    discountsCost: '$12.30'
  },
  {
    id: 3,
    title: 'Gun: Good Weapon2 1.2',
    cost: '$11.10 FN',
    discountsCost: '$12.30'
  },
  {
    id: 4,
    title: 'Gun: Good Weapon2 1.2',
    cost: '$11.10 FN',
    discountsCost: '$12.30'
  },
  {
    id: 5,
    title: 'Gun: Good Weapon2 1.2',
    cost: '$11.10 FN',
    discountsCost: '$12.30'
  },
  {
    id: 6,
    title: 'Gun: Good Weapon2 1.2',
    cost: '$11.10 FN',
    discountsCost: '$12.30'
  },
  {
    id: 7,
    title: 'Gun: Good Weapon2 1.2',
    cost: '$11.10 FN',
    discountsCost: '$12.30'
  },
  {
    id: 8,
    title: 'Gun: Good Weapon2 1.2',
    cost: '$11.10 FN',
    discountsCost: '$12.30'
  },
  {
    id: 9,
    title: 'Gun: Good Weapon2 1.2',
    cost: '$11.10 FN',
    discountsCost: '$12.30'
  },
  {
    id: 10,
    title: 'Gun: Good Weapon2 1.2',
    cost: '$11.10 FN',
    discountsCost: '$12.30'
  },
  {
    id: 11,
    title: 'Gun: Good Weapon2 1.2',
    cost: '$11.10 FN',
    discountsCost: '$12.30'
  },
  {
    id: 12,
    title: 'Gun: Good Weapon2 1.2',
    cost: '$11.10 FN',
    discountsCost: '$12.30'
  }
]

export default compose(
  withProps(() => ({ skins: tmpArrSkins })),
  pure
)(AsyncSkinsDesktop)
