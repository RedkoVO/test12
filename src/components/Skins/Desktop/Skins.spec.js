import * as React from 'react'
import renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router-dom'

import Skins from './'

it('renders correctly', () => {
  const tree = renderer
    .create(
      <MemoryRouter keyLength={0}>
        <Skins handleLogout={() => {}} isDesktop={false} />
      </MemoryRouter>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
