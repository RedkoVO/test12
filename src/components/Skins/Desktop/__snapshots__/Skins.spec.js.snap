// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="mainWrapperTrade"
>
  <div
    className="Skins-containerTradeWrap-1"
  >
    <div
      className="Skins-containerTrade-2"
    >
      <header
        className="Skins-headerSkins-3"
      >
        <ul
          className="MenuNavigation-root-5 dark"
        >
          <li
            className={undefined}
          >
            <div
              className="MenuNavigation-logout-14"
              onClick={[Function]}
            >
              Logout
            </div>
          </li>
          <li>
            <a
              className="MenuNavigation-headerMenuItem-6"
              href="/"
              onClick={[Function]}
            >
              <i
                className="MenuNavigation-menuIcon-7 MenuNavigation-dashboard-8"
              />
              <br />
               Dashboard
            </a>
          </li>
          <li>
            <a
              className="MenuNavigation-headerMenuItem-6"
              href="/"
              onClick={[Function]}
            >
              <i
                className="MenuNavigation-menuIcon-7 MenuNavigation-walletIcon-9"
              />
              <br />
               Wallet
            </a>
          </li>
          <li>
            <a
              className="MenuNavigation-headerMenuItem-6"
              href="/"
              onClick={[Function]}
            >
              <i
                className="MenuNavigation-menuIcon-7 MenuNavigation-shop-10"
              />
              <br />
               Shop
            </a>
          </li>
          <li>
            <a
              className="MenuNavigation-headerMenuItem-6"
              href="/games"
              onClick={[Function]}
            >
              <i
                className="MenuNavigation-menuIcon-7 MenuNavigation-games-11"
              />
              <br />
               Games
            </a>
          </li>
          <li>
            <a
              className="MenuNavigation-headerMenuItem-6"
              href="/"
              onClick={[Function]}
            >
              <i
                className="MenuNavigation-menuIcon-7 MenuNavigation-friends-12"
              />
              <br />
               Friends
            </a>
          </li>
          <li>
            <a
              className="MenuNavigation-headerMenuItem-6"
              href="/"
              onClick={[Function]}
            >
              <i
                className="MenuNavigation-menuIcon-7 MenuNavigation-settings-13"
              />
              <br />
               Settings
            </a>
          </li>
        </ul>
      </header>
      <main
        className="Skins-tradeMain-4"
      >
        <div
          className="TradeTop-root-15"
        >
          <div
            className="TradeTop-tradeGame-16"
          >
            <div
              className="TradeTop-tradeGameSelect-17"
            >
              Game:
            </div>
            <div
              className="TradeTop-tradeGameItem-18"
            >
              CS:GO
            </div>
          </div>
          <div
            className="TradeTop-tradeStream-19"
          >
            <div
              className="TradeTop-tradeStreamTitle-20"
            >
              Put your steam trade url: 
            </div>
            <div
              className="TradeTop-tradeStreamUrlWrap-21"
            >
              <input
                className="TradeTop-tradeStreamUrl-22"
                placeholder="https://steamcommunity.com/groups/dota2lounge/discussions/0/558"
                type="text"
              />
            </div>
          </div>
        </div>
        <div
          className="TradeSkins-root-23"
        >
          <div
            className="TradeSkins-tradeSkinsTitle-24"
          >
            Select skins you want to sale:
          </div>
          <div
            className="TradeSkins-tradeSkinsItemWrap-25"
          >
            <div
              className="TradeSkins-tradeSkinsItem-26 TradeSkins-skinSelected-27"
            >
              <div
                className="TradeSkins-skinsItemImg-28"
              >
                <img
                  alt=""
                  src="gun.png"
                />
              </div>
              <div
                className="TradeSkins-skinsItemName-29"
              >
                Gun: Good Weapon2 1.2
              </div>
              <div
                className="TradeSkins-skinsItemPrice-30"
              >
                $11.10 FN
              </div>
              <div
                className="TradeSkins-skinsItemSale-31"
              >
                $12.30
              </div>
            </div>
            <div
              className="TradeSkins-tradeSkinsItem-26 TradeSkins-skinSelected-27"
            >
              <div
                className="TradeSkins-skinsItemImg-28"
              >
                <img
                  alt=""
                  src="gun.png"
                />
              </div>
              <div
                className="TradeSkins-skinsItemName-29"
              >
                Gun: Good Weapon2 1.2
              </div>
              <div
                className="TradeSkins-skinsItemPrice-30"
              >
                $11.10 FN
              </div>
              <div
                className="TradeSkins-skinsItemSale-31"
              >
                $12.30
              </div>
            </div>
            <div
              className="TradeSkins-tradeSkinsItem-26 TradeSkins-skinSelected-27"
            >
              <div
                className="TradeSkins-skinsItemImg-28"
              >
                <img
                  alt=""
                  src="gun.png"
                />
              </div>
              <div
                className="TradeSkins-skinsItemName-29"
              >
                Gun: Good Weapon2 1.2
              </div>
              <div
                className="TradeSkins-skinsItemPrice-30"
              >
                $11.10 FN
              </div>
              <div
                className="TradeSkins-skinsItemSale-31"
              >
                $12.30
              </div>
            </div>
            <div
              className="TradeSkins-tradeSkinsItem-26 TradeSkins-skinSelected-27"
            >
              <div
                className="TradeSkins-skinsItemImg-28"
              >
                <img
                  alt=""
                  src="gun.png"
                />
              </div>
              <div
                className="TradeSkins-skinsItemName-29"
              >
                Gun: Good Weapon2 1.2
              </div>
              <div
                className="TradeSkins-skinsItemPrice-30"
              >
                $11.10 FN
              </div>
              <div
                className="TradeSkins-skinsItemSale-31"
              >
                $12.30
              </div>
            </div>
            <div
              className="TradeSkins-tradeSkinsItem-26 TradeSkins-skinSelected-27"
            >
              <div
                className="TradeSkins-skinsItemImg-28"
              >
                <img
                  alt=""
                  src="gun.png"
                />
              </div>
              <div
                className="TradeSkins-skinsItemName-29"
              >
                Gun: Good Weapon2 1.2
              </div>
              <div
                className="TradeSkins-skinsItemPrice-30"
              >
                $11.10 FN
              </div>
              <div
                className="TradeSkins-skinsItemSale-31"
              >
                $12.30
              </div>
            </div>
            <div
              className="TradeSkins-tradeSkinsItem-26 TradeSkins-skinSelected-27"
            >
              <div
                className="TradeSkins-skinsItemImg-28"
              >
                <img
                  alt=""
                  src="gun.png"
                />
              </div>
              <div
                className="TradeSkins-skinsItemName-29"
              >
                Gun: Good Weapon2 1.2
              </div>
              <div
                className="TradeSkins-skinsItemPrice-30"
              >
                $11.10 FN
              </div>
              <div
                className="TradeSkins-skinsItemSale-31"
              >
                $12.30
              </div>
            </div>
            <div
              className="TradeSkins-tradeSkinsItem-26 TradeSkins-skinSelected-27"
            >
              <div
                className="TradeSkins-skinsItemImg-28"
              >
                <img
                  alt=""
                  src="gun.png"
                />
              </div>
              <div
                className="TradeSkins-skinsItemName-29"
              >
                Gun: Good Weapon2 1.2
              </div>
              <div
                className="TradeSkins-skinsItemPrice-30"
              >
                $11.10 FN
              </div>
              <div
                className="TradeSkins-skinsItemSale-31"
              >
                $12.30
              </div>
            </div>
            <div
              className="TradeSkins-tradeSkinsItem-26 TradeSkins-skinSelected-27"
            >
              <div
                className="TradeSkins-skinsItemImg-28"
              >
                <img
                  alt=""
                  src="gun.png"
                />
              </div>
              <div
                className="TradeSkins-skinsItemName-29"
              >
                Gun: Good Weapon2 1.2
              </div>
              <div
                className="TradeSkins-skinsItemPrice-30"
              >
                $11.10 FN
              </div>
              <div
                className="TradeSkins-skinsItemSale-31"
              >
                $12.30
              </div>
            </div>
            <div
              className="TradeSkins-tradeSkinsItem-26 TradeSkins-skinSelected-27"
            >
              <div
                className="TradeSkins-skinsItemImg-28"
              >
                <img
                  alt=""
                  src="gun.png"
                />
              </div>
              <div
                className="TradeSkins-skinsItemName-29"
              >
                Gun: Good Weapon2 1.2
              </div>
              <div
                className="TradeSkins-skinsItemPrice-30"
              >
                $11.10 FN
              </div>
              <div
                className="TradeSkins-skinsItemSale-31"
              >
                $12.30
              </div>
            </div>
            <div
              className="TradeSkins-tradeSkinsItem-26 TradeSkins-skinSelected-27"
            >
              <div
                className="TradeSkins-skinsItemImg-28"
              >
                <img
                  alt=""
                  src="gun.png"
                />
              </div>
              <div
                className="TradeSkins-skinsItemName-29"
              >
                Gun: Good Weapon2 1.2
              </div>
              <div
                className="TradeSkins-skinsItemPrice-30"
              >
                $11.10 FN
              </div>
              <div
                className="TradeSkins-skinsItemSale-31"
              >
                $12.30
              </div>
            </div>
            <div
              className="TradeSkins-tradeSkinsItem-26 TradeSkins-skinSelected-27"
            >
              <div
                className="TradeSkins-skinsItemImg-28"
              >
                <img
                  alt=""
                  src="gun.png"
                />
              </div>
              <div
                className="TradeSkins-skinsItemName-29"
              >
                Gun: Good Weapon2 1.2
              </div>
              <div
                className="TradeSkins-skinsItemPrice-30"
              >
                $11.10 FN
              </div>
              <div
                className="TradeSkins-skinsItemSale-31"
              >
                $12.30
              </div>
            </div>
            <div
              className="TradeSkins-tradeSkinsItem-26 TradeSkins-skinSelected-27"
            >
              <div
                className="TradeSkins-skinsItemImg-28"
              >
                <img
                  alt=""
                  src="gun.png"
                />
              </div>
              <div
                className="TradeSkins-skinsItemName-29"
              >
                Gun: Good Weapon2 1.2
              </div>
              <div
                className="TradeSkins-skinsItemPrice-30"
              >
                $11.10 FN
              </div>
              <div
                className="TradeSkins-skinsItemSale-31"
              >
                $12.30
              </div>
            </div>
          </div>
          <ul
            className="TradeSkins-pageNavi-32"
          >
            <li>
              <div
                className="TradeSkins-previos-33"
              >
                Prev
              </div>
            </li>
            <li>
              <div
                className={undefined}
              >
                1
              </div>
            </li>
            <li>
              <div>
                2
              </div>
            </li>
            <li>
              <div
                className="TradeSkins-next-34"
              >
                Next
              </div>
            </li>
          </ul>
        </div>
        <div
          className="TradePayment-root-35"
        >
          <div
            className="TradePayment-paymentTitle-36"
          >
            Choose the way to cash out:
          </div>
          <div
            className="TradePayment-paymentWay-37"
          >
            <div
              className="TradePayment-paymentWayItem-38 TradePayment-paymentSelectItem-39"
            >
              <div
                className="TradePayment-paymentWayItemImg-40"
              >
                <img
                  alt=""
                  src="visa.png"
                />
              </div>
              <div
                className="TradePayment-paymentWayItemName-41"
              >
                Credit Card
              </div>
              <div
                className="TradePayment-paymentWayItemUnderline-42 TradePayment-selected-43"
              />
            </div>
            <div
              className="TradePayment-paymentWayItem-38"
            >
              <div
                className="TradePayment-paymentWayItemImg-40"
              >
                <img
                  alt=""
                  src="bitcnpay.png"
                />
              </div>
              <div
                className="TradePayment-paymentWayItemName-41"
              >
                Bitcoin
              </div>
              <div
                className="TradePayment-paymentWayItemUnderline-42"
              />
            </div>
            <div
              className="TradePayment-paymentWayItem-38"
            >
              <div
                className="TradePayment-paymentWayItemImg-40"
              >
                <img
                  alt=""
                  src="agspay.png"
                />
              </div>
              <div
                className="TradePayment-paymentWayItemName-41"
              >
                CryptaurCoin
              </div>
              <div
                className="TradePayment-paymentWayItemUnderline-42"
              />
            </div>
          </div>
          <div
            className="PaymentCard-root-54"
          >
            <div
              className="PaymentCard-paymentCardTitle-55"
            >
              Cashout on my Credit card:
            </div>
            <div
              className="PaymentCard-paymentInputWrap-56"
            >
              <input
                className="PaymentCard-paymentCardNum-58"
                placeholder="Card Number"
                type="text"
              />
            </div>
            <div
              className="PaymentCard-paymentInputWrap2-57"
            >
              <input
                className="PaymentCard-paymentCardHolder-59"
                placeholder="Card Holder"
                type="text"
              />
            </div>
            <a
              className="PaymentCard-paymentCardMonth-60"
              href="#"
            >
              MM
            </a>
            <a
              className="PaymentCard-paymentCardYear-61"
              href="#"
            >
              YYYY
            </a>
            <div
              className="PaymentCard-paymentInputWrap3-62"
            >
              <input
                className="PaymentCard-paymentCardCvc-63"
                placeholder="CVC"
                type="text"
              />
            </div>
          </div>
          <div
            className="PaymentSelected-root-64"
          >
            <div
              className="PaymentSelected-paymentSelectedTitle-65"
            >
              You selected 6 items:
            </div>
            <div
              className="PaymentSelected-paymentSelectedItem-66"
            >
              <div
                className={undefined}
              >
                0
                ) 
                Good EWeapon 1.2
                 ....
              </div>
              <div
                className="PaymentSelected-selectedPrice-67"
              >
                $12.20
              </div>
              <a
                className="PaymentSelected-selectedClose-68"
                href="#"
              />
            </div>
            <div
              className="PaymentSelected-paymentSelectedItem-66"
            >
              <div
                className={undefined}
              >
                1
                ) 
                Good EWeapon 1.2
                 ....
              </div>
              <div
                className="PaymentSelected-selectedPrice-67"
              >
                $12.20
              </div>
              <a
                className="PaymentSelected-selectedClose-68"
                href="#"
              />
            </div>
            <div
              className="PaymentSelected-paymentSelectedItem-66"
            >
              <div
                className={undefined}
              >
                2
                ) 
                Good EWeapon 1.2
                 ....
              </div>
              <div
                className="PaymentSelected-selectedPrice-67"
              >
                $12.20
              </div>
              <a
                className="PaymentSelected-selectedClose-68"
                href="#"
              />
            </div>
            <div
              className="PaymentSelected-paymentSelectedItem-66"
            >
              <div
                className={undefined}
              >
                3
                ) 
                Good EWeapon 1.2
                 ....
              </div>
              <div
                className="PaymentSelected-selectedPrice-67"
              >
                $12.20
              </div>
              <a
                className="PaymentSelected-selectedClose-68"
                href="#"
              />
            </div>
            <div
              className="PaymentSelected-paymentSelectedItem-66"
            >
              <div
                className={undefined}
              >
                4
                ) 
                Good EWeapon 1.2
                 ....
              </div>
              <div
                className="PaymentSelected-selectedPrice-67"
              >
                $12.20
              </div>
              <a
                className="PaymentSelected-selectedClose-68"
                href="#"
              />
            </div>
            <div
              className="PaymentSelected-paymentSelectedItem-66"
            >
              <div
                className={undefined}
              >
                5
                ) 
                Good EWeapon 1.2
                 ....
              </div>
              <div
                className="PaymentSelected-selectedPrice-67"
              >
                $12.20
              </div>
              <a
                className="PaymentSelected-selectedClose-68"
                href="#"
              />
            </div>
          </div>
          <div
            className="TradePayment-paymentAmmount-44"
          >
            <div
              className="TradePayment-paymentAmmountText-45"
            >
              Total ammount: 
            </div>
            <div
              className="TradePayment-paymentAmmountFill-46"
            />
            <div
              className="TradePayment-paymentAmmountPrice-47"
            >
              $67.45
            </div>
          </div>
          <div
            className="TradePayment-paymentBonus-48"
          >
            <input
              className="TradePayment-paymentBonusCheck-49"
              id="paymentBonusCheck"
              type="checkbox"
            />
            <label
              className="TradePayment-paymentBonusCheckLabel-50"
              htmlFor="paymentBonusCheck"
            >
              Yes, I have a bonus code!
            </label>
          </div>
          <div
            className="TradePayment-paymentInputWrap4-51"
          >
            <input
              className="TradePayment-paymentBonusInput-52"
              placeholder="Enter Your Bonus Code"
              type="text"
            />
          </div>
          <a
            className="TradePayment-paymentBtn-53"
            href="#"
          >
            CASHOUT
          </a>
        </div>
      </main>
    </div>
  </div>
  <footer
    className="Footer-root-69"
  >
    <ul
      className="Footer-footerNav-70"
    >
      <li>
        <a
          className="Footer-footerNavItem-71"
          href="/"
          onClick={[Function]}
        >
          Dashboard
        </a>
      </li>
      <li>
        <a
          className="Footer-footerNavItem-71"
          href="/"
          onClick={[Function]}
        >
          Wallet
        </a>
      </li>
      <li>
        <a
          className="Footer-footerNavItem-71"
          href="/"
          onClick={[Function]}
        >
          Shop
        </a>
      </li>
      <li>
        <a
          className="Footer-footerNavItem-71"
          href="/games"
          onClick={[Function]}
        >
          Games
        </a>
      </li>
      <li>
        <a
          className="Footer-footerNavItem-71"
          href="/"
          onClick={[Function]}
        >
          Friends
        </a>
      </li>
      <li>
        <a
          className="Footer-footerNavItem-71"
          href="/"
          onClick={[Function]}
        >
          Settings
        </a>
      </li>
    </ul>
    <span
      className={undefined}
    >
      Vault. All Rights Reserved 2018
    </span>
  </footer>
</div>
`;
